lib/index.js

```diff
@@ -297,7 +298,9 @@ Runner.prototype = {
     // Keep around N most recent build directories.
     // Default is 0, ie wipe at start of each run.
     // Later, this can be configurable in the UI.
-    keeper({baseDir: path.join(this.config.dataDir, "data"), count: 0}, function(err) {
+    keeper({baseDir: path.join(this.config.dataDir, "data"), count: this.config.concurrentJobs - 1}, function(err) {
```

Comment:
I realized after I put this together that this is too simplistic; it assumes that, at most, the most recent n-1 data directories (possibly) belong to concurrent builds. If you have one long build and many short ones that happen before it completes, you can have many more data directories still in use.

I'll have to introduce a better way to track which data directories are in use and which are not.